CMSgov/design-system
# [WNMGDS-1376] initial add of gatsby doc site

## Summary

Adding a new Gatsby-powered documentation site at `packages/docs`

I added some placeholder components and some styles to help establish a directory structure.

## How to test

* `yarn install`
* ~navigate to packages/docs and `yarn develop` to run Gatsby locally~ run `yarn start:gatsby` from the root directory. This should spin up localhost:8000
* You should see a simple page with header, main & footer text

## Notes
* the tsconfig is generated by Gatsby. If there are updates or a way to make it consistent with our larger system, please let me know
* I got our core styles imported and working (with fonts!). This section of code may update if/when we get a theme switcher
